config/retry.go

```diff
@@ -13,6 +13,9 @@ const (
 	// DefaultRetryBackoff is the default base for the exponential backoff
 	// algorithm.
 	DefaultRetryBackoff = 250 * time.Millisecond
+
+	// DefaultRetryMaxBackoff is the default maximum of backoff time
+	DefaultRetryMaxBackoff = 0 * time.Millisecond
```

Comment:
Yeah I would change all the defaults. They are too short currently in my opinion.

I would do:
```
DefaultRetryAttempts = 12
DefaultRetryBackoff = 250 * time.Milliseconds
DefaultRetryMaxBackoff = 1 * time.Minute
```

Leads to:

| attempt 	| sleep 	| total 	|
|---------	|-------	|-------	|
| 1       	| .25s  	| .25s  	|
| 2       	| .5s   	| .75s  	|
| 3       	| 1s    	| 1.75s 	|
| 4       	| 2s    	| ~4s   	|
| 5       	| 4s    	| ~8s   	|
| 6       	| 8s    	| ~16s  	|
| 7       	| 16s   	| ~32s  	|
| 8       	| 32s   	| ~1m   	|
| 9       	| ~1m   	| ~2m   	|
| 10      	| ~1m   	| ~3m   	|
| 11      	| ~1m   	| ~4m   	|
| 12      	| ~1m   	| ~5m   	|
| ...     	| ...   	| ...   	|

Gives you five minutes of retrying with a good back off ramp and fast early retries.